src/test/scripts/check_enterprise_merge.sh

```diff
@@ -4,52 +4,55 @@
 set -u
 # exit immediately if a command fails
 set -e
+# list executed commands
+set -x
 
-# try_merge sees if we can merge "from" branch to "to" branch
-# it will exit with nonzero code if the merge fails because of conflicts.
-try_merge() {
-    to=$1
-    from=$2
-    git checkout "${to}"
-    # this will exit since -e option is set and it will return non-zero code on conflicts.
-    git merge --no-ff --no-commit "${from}"
-
-}
+PR_BRANCH="${CIRCLE_BRANCH}"
 
-cd /tmp
-if [ ! -d citus-enterprise ]; then
-    git clone https://${GIT_USERNAME}:${GIT_TOKEN}@github.com/citusdata/citus-enterprise
-fi
+git remote add enterprise "https://${GIT_USERNAME}:${GIT_TOKEN}@github.com/citusdata/citus-enterprise"
```

Comment:
For example, this will be shown in the output of the job, the secrets are filtered out in the output but I still don't feel comfortable about this since it is public.

Reply:
I've moved this before `set -x`